src/ripple/unity/app_paths.cpp

```diff
@@ -45,6 +45,8 @@
 #include <ripple/app/paths/cursor/ReverseLiquidityForAccount.cpp>
 #include <ripple/app/paths/cursor/RippleLiquidity.cpp>
 #include <ripple/app/paths/RippleLineCache.cpp>
+
+#include <ripple/app/paths/tests/Path.test.cpp>
```

Comment:
This `#include` is in both this file and src/ripple/unity/app3.cpp. I don't think it's kosher to have the same file in two unity files.